shared/actions/search.js

```diff
@@ -223,3 +205,27 @@ export function reset (): Reset {
     payload: {},
   }
 }
+
+function * handleSearchRequest (action: Search): SagaGenerator<any, any> {
+  if (action.error) {
+    return
+  }
+
+  const {term, platform} = action.payload
+  try {
+    const results: APIRes = yield * cpsWithWaiting(waiting, _search, term, platform)
+    const json = JSON.parse(results.body)
+    const followingState = yield select((state: TypedState) => state.config.following) // eslint-disable-line arrow-parens
```

Comment:
Ah I see, the `select` has to happen somewhere near here since this runs in response to the action.